.github/workflows/main.yml

```diff
@@ -23,6 +23,10 @@ jobs:
       - name: Install dependencies
         run: |
           yarn install --frozen-lockfile
+          # Fix for: https://github.com/actions/runner-images/issues/675
+          sudo gem install apt-spy2
+          sudo apt-spy2 fix --commit --launchpad --country=US
+          sudo apt-get update
```

Comment:
@jerivas This look okay? If so, I might add the same fix to anchor-pos and popover polyfills...

Reply:
LGTM 🚀